### DISCUSSION TITLE
Logging Web Traffic as Structured Events from Apache HTTP Server 2.4

### DISCUSSION THREAD

**Discussion by n1njab0b** [datalust/seq-tickets#1787] [Help] [answered]
I would love to be able to easily log all the HTTP traffic coming in to my Apache HTTP server.  
As far as I can tell, the best options include:

- Streaming events with syslog (not structured)
- Parsing log files with seqcli (requires batch processing)
- Streaming log files with Apache Flume (requires separate application install)

I'm surprised though with the difficulty I have had trying to find any guidance on how best to do this.  Is it not a common practice to send Apache HTTP event data to Seq?

**Comment by liammclennan** [comment, marked as answer]
Hi @n1njab0b 

Another option is [parsing log files with Fluent Bit](https://blog.datalust.co/tailing-a-log-file-or-folder-with-fluent-bit-and-seq/).

Seq doesn't have built-in support for parsing text logs to structured data, so any option will require an extra step.
